schemas/rum/_vital-info-schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "rum/_vital-info-schema.json",
+  "title": "RumVitalInfo",
+  "type": "object",
+  "description": "Schema of properties for a vital",
+  "required": ["view_min", "view_max", "view_average"],
+  "properties": {
+    "view_min": {
+      "type": "number",
+      "description": "The minimum value seen for this vital in the view.",
```

Comment:
Nitpick: I'd replace `in the view` by `during this view's lifetime`
Same applies for `view_max` and `view_average`